### ISSUE TITLE
Use strings?

### ISSUE THREAD

**Issue by joshgoebel** [PaulBGD/PixelFont#5]
You could get even tighter if you just used strings:

```js
      'Z': [
            "11111",
            "...1.",
            "..1..",
            ".1...",
            "11111",
        ],
```

**Comment by PaulBGD** [comment]
Yep there's a lot of improvements that could be made! I think the most significant one would be representing them as a number, as you can easily store all 25 bits as an integer. Feel free to mess with this all you want!

**Comment by joshgoebel** [comment]
> I think the most significant one would be representing them as a number, as you can easily store all 25 bits as an integer. 

Oh sure there are ways to encode them such that they are smaller and no longer easily readable/hackable... but I assumed that wasn't "in scope"... with strings you get the benefits of easy to edit, but also save space.  If you just want them as small as possible at some point it starts to make sense to just bundle them in a png/gift sprite sheet, etc.

**Comment by PaulBGD** [comment]
The main reason I use an array for this is just because it's very easy to visualize for the article, although you're right that strings are very good too.

**Comment by joshgoebel** [comment]
Yeah, you'd still have the visualize (easy to edit) but with less space usage.  Should compress better because you remove all the delimiters and such.

**Comment by ASXploit** [comment]
I have a similar code i have been working on. A friend created it as an idea a few years ago but gave up on it after he kind of got it working.  I went through it,  updated it and ported over to ES6 where I'm still introducing updates.  However I wish there was a better way to create alphabetical letters,  instead of storing these long ugly arrays in the code. I use them for images like game sprites which is fine but to do the whole alphabet lol

**Comment by joshgoebel** [comment]
At some point you just stop writing code at all and write (or find) a tool to convert raw sprite/font sheets into code for you.  Then you never look at the code, you just use a pixel editor... much nicer.
